spec/lib/api/v3/work_packages/work_package_representer_spec.rb

```diff
@@ -426,12 +418,22 @@
       end
 
       describe 'schema' do
-        let(:schema_id) { "#{work_package.project.id}-#{work_package.type.id}" }
-        let(:link) { "/api/v3/work_packages/schemas/#{schema_id}".to_json }
-        let(:href_path) { '_links/schema/href' }
+        it_behaves_like 'has an untitled link' do
+          let(:link) { 'schema' }
+          let(:href) {
+            api_v3_paths.work_package_schema(work_package.project.id, work_package.type.id)
+          }
+        end
+      end
 
-        it 'has a link' do
-          is_expected.to be_json_eql(link).at_path(href_path)
+      describe 'attachments' do
+        it_behaves_like 'has an untitled link' do
+          let(:link) { 'attachments' }
+          let(:href) { api_v3_paths.attachments_by_work_package(work_package.id) }
+        end
+
+        it 'embeds the watchers as collection' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.